css/style.css

```diff
@@ -1425,12 +1425,29 @@ body#Home #content > .intro .download .btn.action:hover
 
 .org_list
 {
+    max-width: 1400px;
+    margin: 0 auto;
+    padding: 10px 0;
+
+    display: -webkit-box;
+    display: -ms-flexbox;
     display: flex;
+
+    -webkit-box-align: center;
+    -ms-flex-align: center;
     align-items: center;
+
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
```

Comment:
It's a standard now: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction